### PR TITLE
Open source resources

### PR DESCRIPTION
**Pull request by hvwaldow**
We should use the term "Open Source" correctly. In our community that commonly refers to a set of licenses for software, not to teaching material.

I moved that paragraph about continually updated online-material (what is really meant here) to the top of the section.

The proper heading for what is following (which needs more work) could be "Various resources".

Also the work of Fogel (2005 and 2017) totally is not "the reference work on RSE", so I removed that sentence.

### CODE REVIEW

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -559,7 +566,7 @@ an important role, even if they do not necessarily touch much of the code themse
 RSEs who focus on teaching the next generation of researchers and/or RSEs play
 a vital role in quality research software.
 
-#### ${DOMAIN}-RSE
```
Comment (CaptainSifff) [inline paper_teaching-learning-RSE.md:562]: no bashisms?

**Comment by CaptainSifff** [comment]
here's the link to our old pad for that: https://pad.gwdg.de/teaching-RSE-resource-pad#

I wonder whether you want to open the discussion to include some institution?

**Comment by jpthiele** [comment]
In the nomenclature of Open Science out would rather be Open Educational Resources, right?
So then maybe we should use that established term if we want to highlight it. 
But this way it also looks fine to me.

File: paper_teaching-learning-RSE.md (hunk 1 above)
Comment (hvwaldow) [reply paper_teaching-learning-RSE.md:562]: This bashism throws off my Mardown highlighting, so I tend to remove it locally and then forget to change it back.

**Comment by jngrad** [comment]
@hvwaldow The section you moved to the appendix changed quite a bit in recently merged PRs, with e.g. a discussion on databases of online resources. I did my best to resolve the merge conflicts in cf9206854ae0993f17181c096ce4f6d5e2894831. Feel free to fast-forward my merge commit and continue working from there (`git merge --ff-only upstream/open_source_resources`).

**Comment by hvwaldow** [comment]
I split the section "RSE-related training ressources" into 2 parts:
1. Short descriptions of organizations & initiatives that we hope are somewhat persistent and have a track-record for providing workshops, development of training material, .. that is very close to RSE core-skills. Such as Software Carpentry.
2. We have a long list of valuable yet hard to sort material from various sources that might well fall out of maintenance, become outdated, or disappear. I think we should put such a list in an appendix. 

Status now is:  1. is good (for me), but still might get new entries when I work through 2., which is currently all text-fragments ("FIXME" and whatnot) that I have thrown under the Appendix "Sources for RSE-related training material".

I intend to have that in Merge-ready shape by next meeting.

Feel free to drop any orgas / initiatives & materials here in the comments, and I'll include them.

I do think a bit of quality threshold is warranted, so I have for example removed a resource that provides just a list like ours (other people's material), but worse.

**Comment by hvwaldow** [comment]
Replaced with #86 for better collaboration.

**Pull request closed without merging**